src/MICore/Debugger.cs

```diff
@@ -492,8 +494,6 @@ public async Task WaitForConsoleDebuggerInitialize(CancellationToken token)
             {
                 _consoleDebuggerInitializeCompletionSource = null;
 
-                // We no longer care about keeping these, so empty them out
-                _initializationLog = null;
```

Comment:
You want to put this back

Reply:
I was in two minds on this one since the logs weren't too long and I wasn't sure if we really wanted to clear them. But given we could fit our logic at the right place (thanks for the tip), and it was cleared previously, I assume it must have been done for a reason. So reverting to the old state!